docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlowTransform.cs

```diff
@@ -0,0 +1,93 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Runtime.Data;
+using System;
+using System.Linq;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    class TensorFlowTransformExample
+    {
+        /// <summary>
+        /// Example use of the TensorFlowEstimator in a ML.NET pipeline.
+        /// </summary>
+        public static void TensorFlowScoringSample()
+        {
+            // Download the ResNet 101 model from the location below.
+            // https://storage.googleapis.com/download.tensorflow.org/models/tflite_11_05_08/resnet_v2_101.tgz
+            var modelLocation = @"resnet_v2_101/resnet_v2_101_299_frozen.pb";
```

Comment:
Maybe just preference, but I would put `@"Dynamic/resnet_v2_101/resnet_v2_101_299_frozen.pb`.  I dropped the model into the same directory as the source file (and set "copy to output"), but that doesn't work by default because visual studio moves the binaries outside Dynamic in the bin.  Adding "Dynamic" to the path makes it work by default, but maybe it isn't necessary for a sample to demonstrate the API.

Reply:
For specifying path to model file, I am following same pattern as the OnnxTransform. (prefer to keep both consistent)

---
In reply to: [239241951](https://github.com/dotnet/machinelearning/pull/1812#discussion_r239241951) [](ancestors = 239241951)

Reply:
Ah, sounds good.


---
In reply to: [239551754](https://github.com/dotnet/machinelearning/pull/1812#discussion_r239551754) [](ancestors = 239551754,239241951)